code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/config/GithubConfig.java

```diff
@@ -0,0 +1,38 @@
+package io.cattle.platform.iaas.api.auth.config;
+
+import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+import java.util.List;
+
+@Type(name = "githubconfig")
+public class GithubConfig {
+
+    private Boolean enabled;
+    private String clientId;
+    private List<String> allowUsers;
```

Comment:
we want "allowUsers", not "allowedUsers?"  @vincent99 Do you care?  The english sounds wrong there.